assets/src/edit-story/components/inspector/karma/inspectorTabs.karma.js

```diff
@@ -134,5 +134,36 @@ describe('Inspector Tabs integration', () => {
         ).not.toBe(null);
       });
     });
+
+    it('should navigate to checklist tab after following "review checklist" button in dialog on publishing story', async () => {
+      fixture.events.click(fixture.editor.titleBar.publish);
+      // Ensure the debounced callback has taken effect.
+      await fixture.events.sleep(800);
+
+      const reviewButton = await fixture.screen.getByRole('button', {
+        name: /^Review Checklist$/,
+      });
+      await fixture.events.click(reviewButton);
+      await fixture.events.sleep(300);
```

Comment:
Do we need to sleep here if we're already waiting for disabled to change on the next lines?